examples/FasterRCNN/data.py

```diff
@@ -345,7 +345,10 @@ def get_train_dataflow():
     # Filter out images that have no gt boxes, but this filter shall not be applied for testing.
     # The model does support training with empty images, but it is not useful for COCO.
     num = len(roidbs)
-    roidbs = list(filter(lambda img: len(img["boxes"][img["is_crowd"] == 0]) > 0, roidbs))
+    if cfg.DATA.FILTER_EMPTY_ANNOTATIONS:
+        roidbs = list(filter(lambda img: len(img["boxes"][img["is_crowd"] == 0]) >= 0, roidbs))
+    else:
+        roidbs = list(filter(lambda img: len(img["boxes"][img["is_crowd"] == 0]) > 0, roidbs))        
```

Comment:
```suggestion
    if cfg.DATA.FILTER_EMPTY_ANNOTATIONS:
        roidbs = list(filter(lambda img: len(img["boxes"][img["is_crowd"] == 0]) > 0, roidbs))
```

`FILTER_EMPTY_ANNOTATIONS=False` means nothing need to be done